core/arch/arm/plat-zynqmp/drivers/puf.c

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2021 Foundries.io Ltd.
+ *
+ * Jorge Ramirez-Ortiz <jorge@foundries.io>
+ */
+
+#include <initcall.h>
+#include <io.h>
+#include <kernel/delay.h>
+#include <mm/core_memprot.h>
+
+#include "puf.h"
+
+/* CSU */
+#define ISR				0x20
+#define PUF_ISR_ACC_ERROR_MASK          BIT(12)
+
+/* PUF */
+#define CSU_PUF_BASE			(CSU_BASE + 0x4000)
+#define CSU_PUF_SIZE			0x1000
+
+#define CMD				0x00
```

Comment:
These are a bit vague names.

For reference the public register map is here:
https://www.xilinx.com/html_docs/registers/ug1087/ug1087-zynq-ultrascale-registers.html

These PUF registers are part of CSU module. Official name for CMD is `puf_cmd`.

Please be a bit more verbose with the name something like:

`PUF_CMD_REG_OFFSET` or so with or without `CSU_` prefix.

Reply:
yes, btw thanks for those pointers to hyperlinks but I know them all really :)

Reply:
@jforissier anyone wants to comment on this? I dislike long names for registers because -for me- it makes the code a PIA to read. I'd rather keep it as is unless what @vesajaaskelainen is a defacto rule now.

Reply:
Well I like both short identifiers, and identifiers that are documented in the official documentation. In case both are incompatible, the context matters. Here the scope of the macros is limited to the file so I'd say we can consider that Zynq/CSU/PUF are implicit and therefore keep shorter names. In a header file things would be a bit different.